src/Holding/ViewHolding/HoldingBoundWith/useBoundWithHoldings.test.js

```diff
@@ -31,9 +31,12 @@ describe('useBoundWithHoldings', () => {
   it('should fetch bound-with holdings', async () => {
     const boundWithItems = [{ hrid: 'BW-ITEM-1', holdingsRecordId: '9e8dc8ce-68f3-4e75-8479-d548ce521157' }];
 
-    const { result, waitFor } = renderHook(() => useBoundWithHoldings(boundWithItems), { wrapper });
+    let result;
 
-    await waitFor(() => !result.current.isLoading);
+    await act(async () => {
+      const hookResult = renderHook(() => useBoundWithHoldings(boundWithItems), { wrapper });
+      result = hookResult.result;
+    });
```

Comment:
You can still use `waitFor` but the implementation has changed a bit. e.g. see [test updates in stripes-core](https://github.com/folio-org/stripes-core/pull/1330/files#diff-ff4f315078c3b7d3f2b30c669e1f2765bbb37027490827750bead83acc5bf1fc)

Reply:
Yes thank you! This one also works: `await act(async () => !result.current.isLoading);`